packages/apm/test/hub.test.ts

```diff
@@ -11,6 +11,28 @@ describe('Hub', () => {
     jest.useRealTimers();
   });
 
+  describe('getTransaction', () => {
```

Comment:
These tests are not technically "correct". defined/undefined test should be moved to `scope.test.ts`, as this functionality is owned by the `Scope`, and tests below should only assert that a given method was called on the scope. `expect(s.getTransaction).toBeCalled()`.

Reply:
Moved to `scope.test.ts`